publication-event-handlers/src/test/java/no/unit/nva/publication/events/doirequests/DoiRequestEventProducerTest.java

```diff
@@ -36,31 +36,33 @@
 
 class DoiRequestEventProducerTest {
 
-    public static final String EVENT_PUBLICATION_WITH_DOI_IS_UPDATED =
-        "resource_update_event_updated_metadata_with_existing_doi.json";
-    public static final String DOI_FIELD = "doi";
-    public static final JsonPointer PUBLICATION_DATA_FIELD = JsonPointer.compile(
+    private static final String DOI_FIELD = "doi";
+    private static final JsonPointer PUBLICATION_DATA_FIELD = JsonPointer.compile(
         "/detail/responsePayload/newPublication");
+    private static final String EVENT_PUBLICATION_WITH_DOI_IS_UPDATED =
+        "doirequests/resource_update_event_updated_metadata_with_existing_doi.json";
     private static final String RESOURCE_UPDATE_EVENT_OLD_AND_NEW_PRESENT_DIFFERENT_NEW_HAS_DOI =
-        "resource_update_event_old_and_new_present_different_new_has_doi.json";
+        "doirequests/resource_update_event_old_and_new_present_different_new_has_doi.json";
```

Comment:
after collecting the handlers in one module, the resources were organized in folders